Basic python and ruby samples

jseldess opened the pull request:
Updating python and ruby samples to exclude the transfer transaction. That'll be in separate samples demonstrating how to retry. 

I test both samples, and they work fine. But I had to comment out the `conn.exec('SET AUTOCOMMIT = ON')` in the ruby file. @tamird, do we not need something like that to make each statement immediately commit?

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/docs/149)
<!-- Reviewable:end -->

tamird commented:
> do we not need something like that to make each statement immediately commit?

No, this is our default. I'm guessing `autocommit=True` happens to be a noop in the python code, so I would try removing that call to set_session.

jseldess commented:
OK, `autocommit=True` is the default in ruby, so I removed that bit. 

Also, @knz, I simplified the clojure sample to mirror what we're doing in the other simple samples. I pasted the transaction bit into txn-sample.clj. Once Andrei's PR is merged, it'd be great if you could look at his go transaction sample and mirror that.

And @nvanbenschoten, I changed `DECIMAL` to `INT` in your node sample, but I'd rather you strip out the transaction stuff yourself and then create a separate example to mirror Andrei's go transaction sample (again, once that's merged).
